### PR TITLE
Create implementation for points

### PR DESCRIPTION
**Pull request by DrewMonroe**
- Create a point class that we can use to represent points
- Renamed the unit tests file because it tests for more than just predicates
- Added vim and emacs to the git ignore

### CODE REVIEW

**Comment by DrewMonroe** [comment]
Created a basic point class, all criticism is welcome. Also looking for input on if/how we should implement slicing for points.

**Comment by DrewMonroe** [comment]
Resolves #5

File: point.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+
+class point:
```
Comment (josephsweeney) [inline point.py]: wtf mate, wheres da caps?

https://www.python.org/dev/peps/pep-0008/#class-names

**Review by josephsweeney — APPROVED** [review]
Seems legit.

**Pull request merged**